src/modules/SettingsRouteDestinations.tsx

```diff
@@ -119,6 +120,8 @@ function SettingsRouteDestinations({ mode }: { mode: NAV_MODE }): React.ReactEle
 
       {CVSettingsRouteDestinations({ mode }).props.children}
 
+      {ProxyKeySettingsRouteDestinations({ mode }).props.children}
```

Comment:
I'd probably make this a wee bit more generic to FF rather than this specific use-case. I don't know what, but we may have more of these settings things in the future that are not proxy related.
```suggestion
      {CFSettingsRouteDestinations({ mode }).props.children}
```

Reply:
Good point, have updated 👍